python/ray/data/tests/test_tfrecords.py

```diff
@@ -455,6 +455,27 @@ def test_read_tfrecords(
     assert np.array_equal(df["string_empty"][0], np.array([], dtype=np.bytes_))
 
 
+@pytest.mark.parametrize("num_cpus", [1, 2, 4])
+def test_read_tfrecords_ray_remote_args(
+    ray_start_cluster,
+    tmp_path,
+    num_cpus,
+):
+    import tensorflow as tf
+
+    cluster = ray_start_cluster
+    cluster.add_node(num_cpus=num_cpus)
```

Comment:
let's follow the other patterns in this PR? (just use `ray_start_regular_shared`)

Reply:
sure, will do.